### PR TITLE
Pro-Bel Cygnus

### PR DESCRIPTION
**Pull request by YongTingLim**
A new branch has been created for SNMP connection for the Pro-Bel Cygnus. Added driver helper to update details on new connection
(previously Smart Serial)

### CODE REVIEW

File: connector/doc/Pro-Bel_Cygnus.md [hunk 1]
```diff
@@ -64,22 +64,85 @@ SERIAL CONNECTION:
...
 
 On this page an overview is given of the crosspoints (**Crosspoints**), the crosspoints buffer (**Crosspoints Buffer**), and status information (**Status Information** and **Communication Status**).
+
+### Version with SNMP Connection
+
+### General
+
+This page shows the general information of the device. It displays its status and name.
+
+### Matrice
```
Comment (MariekeGO) [inline connector/doc/Pro-Bel_Cygnus.md:109]: Could you confirm if "Matrice" is indeed the spelling used in the connector? I'd expect either "Matrices" or "Matrix" to be used – unless the spelling is deliberately different here?

File: connector/doc/Pro-Bel_Cygnus.md [hunk 2]
```diff
@@ -64,22 +64,85 @@ SERIAL CONNECTION:
...
+
+The Audio Modify can also be configured on the table, and its respective enabled and lock state that are used for in the Matrix.
+
+#### Source Association
+
+This page shows an overview of the Source Association Table and the Source Association Sources.
+
+#### Source Settings
+
+This page allows the
```
Comment (MariekeGO) [inline connector/doc/Pro-Bel_Cygnus.md:130]: Something seems to be missing here.

File: connector/doc/Pro-Bel_Cygnus.md [hunk 3]
```diff
@@ -64,22 +64,85 @@ SERIAL CONNECTION:
...
+### Destinations
+
+This page shows an overview of the Destinations Table. The format of the Index is {Levels Index}.{Destinations Id}.\
+The display name can be configured in the format of **Destination Index**.**Choice** which choice can be either the alternative name, 4 character name, 8 character name, 12 character name, or 16 character name.
+
+The routed source can also be configured on the table, and its respective enabled and lock state that are used for in the Matrix.
+
+#### Destination Settings
+
+This page allows the
```
Comment (MariekeGO) [inline connector/doc/Pro-Bel_Cygnus.md:142]: Something seems to be missing here.

File: connector/doc/Pro-Bel_Cygnus.md (hunk 1 above)
Comment (YongTingLim) [reply connector/doc/Pro-Bel_Cygnus.md:109]: Resolved. Spelling should be Matrices

File: connector/doc/Pro-Bel_Cygnus.md (hunk 2 above)
Comment (YongTingLim) [reply connector/doc/Pro-Bel_Cygnus.md:130]: Resolved

File: connector/doc/Pro-Bel_Cygnus.md (hunk 3 above)
Comment (YongTingLim) [reply connector/doc/Pro-Bel_Cygnus.md:142]: Resolved

**Comment by YongTingLim** [comment]
The remarks has been resolved and a new version has been updated.

File: connector/doc/Pro-Bel_Cygnus.md [hunk 4]
```diff
@@ -64,22 +64,85 @@ SERIAL CONNECTION:
...
+
+This page displays the Levels Table. The format of the index is {Matrices Index}.{Levels ID}.
+
+### Sources
+
+This page displays the Sources Table. The format of the index is {Levels Index}.{Sources ID}.
+
+The display name can be configured in the format **Source Index**.**Choice**. This choice can be either the alternative name, a 4-character name, an 8-character name, a 12-character name, or a 16-character name.
+
+The Audio Modify can also be configured on the table, and its respective enabled and lock state that are used for in the Matrix.
```
Comment (MariekeGO) [inline connector/doc/Pro-Bel_Cygnus.md:123]: What is this "Audio Modify" that can be configured? I'd like to be able to rephrase this so it's more clear what exactly the user is configuring here.

File: connector/doc/Pro-Bel_Cygnus.md [hunk 5]
```diff
@@ -64,22 +64,85 @@ SERIAL CONNECTION:
...
+
+#### Source Association
+
+This page displays the Source Association Table and the Source Association Sources.
+
+#### Source Settings
+
+The display name can be configuration of the Sources Table display name. It is formatted in the format of **Source Index**.**Choice** which choice can be either the alternative name, 4 character name, 8 character name, 12 character name, or 16 character name. The default configuration uses the 16 character name choice.
```
Comment (MariekeGO) [inline connector/doc/Pro-Bel_Cygnus.md:131]: This description doesn't really explain to me what the Source Settings page is used for. Could you adjust this?

**Review by MariekeGO — APPROVED** [review]

**Pull request merged**